### PR TITLE
fix compat data for "let" in Chrome

### PR DESCRIPTION
**Pull request by genezx**

### CODE REVIEW

File: javascript/statements.json [hunk 1]
```diff
@@ -1641,10 +1641,12 @@
               "version_added": "41"
             },
             "chrome": {
-              "version_added": "41"
+              "version_added": "49",
+              "notes": "From Chrome 41 to 48 strict mode is required. In Chrome 48 non-strict mode support can be enabled using the flag \"Enable Experimental JavaScript\"."
```
Comment (ExE-Boss) [inline javascript/statements.json]: This should be:
```json
"chrome": [
  {
    "version_added": "49"
  },
  {
    "version_added": "48",
    "version_removed": "49",
    "flags": [
      {
        "type": "preference",
        "name": "Enable Experimental JavaScript Features"
      }
    ],
    "notes": "Support outside of strict mode."
  },
  {
    "version_added": "41",
    "version_removed": "49",
    "partial_implementation": true,
    "notes": "From Chrome 41 to 48 strict mode is required."
  }
]

File: javascript/statements.json [hunk 2]
```diff
@@ -1641,10 +1641,12 @@
...
             },
             "chrome_android": {
-              "version_added": "41"
+              "version_added": "49",
+              "notes": "From Chrome 41 to 48 strict mode is required. In Chrome 48 non-strict mode support can be enabled using the flag \"Enable Experimental JavaScript\"."
```
Comment (ExE-Boss) [inline javascript/statements.json]: ditto

File: javascript/statements.json [hunk 3]
```diff
@@ -1641,10 +1641,12 @@
               "version_added": "41"
```
Comment (ExE-Boss) [inline javascript/statements.json]: This should be:
```json
"chrome": [
  {
    "version_added": "49"
  },
  {
    "version_added": "41",
    "version_removed": "49",
    "partial_implementation": true,
    "notes": "From Chrome 41 to 48 strict mode is required.
  }
]

**Comment by genezx** [comment]
Ping: any progress?

**Pull request merged**